app.py

```
@@ -96,41 +176,44 @@ def check_optional_deps():
-        df_matched, is_matched, matched_treatment_col, matched_covariates
-    )
-
-    # --- 2. Table 1 & Matching Module ---
+    # --- 1. Data Management ---
+    # 🟢 เพิ่ม try-except เพื่อดักจับ Error กรณี tab_data.py ไม่ตรงกัน
+    try:
+        tab_data.data_server("data",
+            df, var_meta, uploaded_file_info,
+            df_matched, is_matched, matched_treatment_col, matched_covariates
+        )
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)